pyomo/contrib/mindtpy/config_options.py

```diff
@@ -575,4 +575,23 @@ def check_config(config):
                 'The threads parameter is corrected to 1 since incumbent callback conflicts with multi-threads mode.')
     if config.solution_pool:
         if config.mip_solver not in {'cplex_persistent', 'gurobi_persistent'}:
+            if config.mip_solver in {'appsi_cplex', 'appsi_gurobi'}:
+                config.logger.info("Solution pool does not support APPSI solver yet.")
             config.mip_solver = 'cplex_persistent'
+    if config.calculate_dual:
```

Comment:
I have some issues with this option's name, as duals are not exactly what we are computing out of the MIP problems, they are the duals of the continuous relaxation fixed at the optimal values of the integer variables (which is slightly different)...

Reply:
Any suggestion for the name?

Reply:
`compute_dual_at_solution` or `calculate_dual_at_solution`